### PR TITLE
Permissions and other fixes

### PR DESCRIPTION
**Pull request by joaojusto**
Permissions:
-Extracted "findUserByEmail" method to server
-Stopped publishing all user data to client
-Only allowed updates on a user by himself

Fixes:
-Corrected "insertIntoTrip" method
-Added the current loged user id to a new trip

### CODE REVIEW

File: client/views/trips/new.coffee [hunk 1]
```diff
@@ -11,6 +11,7 @@ Template.tripsAdd.submit = ->
   Trips.insert
     _id: tripId
     name: name
+  Meteor.call('assignLogedUserToTrip', tripId)
```
Comment (gabrielpoca) [inline client/views/trips/new.coffee]: Why not remove the logic that creates a new trip from here into a method? Simply calling a method like this `Meteor.call('createTrip', name)` I believe would make more sense. What do you think?

File: server/users.coffee [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Meteor.methods
+  findUserByEmail: (email) ->
```
Comment (gabrielpoca) [inline server/users.coffee:2]: Is this method going to be called on the front end? If that's not the case you should [not make it available for the clients to call](http://docs.meteor.com/#methods_header).

File: client/views/trips/new.coffee (hunk 1 above)
Comment (joaojusto) [inline client/views/trips/new.coffee]: I think you are right

File: server/users.coffee (hunk 2 above)
Comment (joaojusto) [inline server/users.coffee:2]: I think it's better to discuss this in person

**Pull request closed without merging**